src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusTriggerExecutor.cs

```diff
@@ -21,10 +22,17 @@ public ServiceBusTriggerExecutor(ITriggeredFunctionExecutor innerExecutor)
         public async Task<FunctionResult> ExecuteAsync(Message value, CancellationToken cancellationToken)
         {
             Guid? parentId = ServiceBusCausalityHelper.GetOwner(value);
+            ServiceBusTriggerDetails triggerDetails = new ServiceBusTriggerDetails
```

Comment:
These changes look good overall, though you might consider not adding these new types and running them through json serialization, since that's a performance hit on every invocation that isn't really needed. The string formatting you had in place before was just as good and more performant.